lib/internal/http2/core.js

```diff
@@ -959,13 +968,14 @@ class Http2Session extends EventEmitter {
       this.on('shutdown', callback);
     }
 
+    const fn = submitShutdown.bind(this, options);
```

Comment:
Could this be a bit more descriptively named? (Similar to `setupFn` elsewhere in this code.)